src/app/asset-page/asset-page.component.scss

```diff
@@ -511,3 +511,67 @@ $titleNavBg: #eee;
 .rights-text {
   text-transform: capitalize;
 }
+
+// Custom styles for dropdown buttons menu
+.show > .btn-primary.dropdown-toggle {
+    border-color: $color-orange-active !important;
+    background-color: $color-orange-active !important;
+}
+
+.dropdown-menu {
+
+    border: 1px solid $color-border-gray;
+    margin-top: 8px;
+    left: 0 !important;
+    min-width: 173px;
+    box-shadow: 0 2px 5px 0 rgba(0, 0, 0, 0.2) !important;
+
+    &.download-menu:before, &.download-menu:after{
+        left: 61% !important;
+    }
+
+    &:before, &:after{
+        content: '';
+        display: block;
+        position: absolute;
+        left: 80%;
+        width: 0;
+        height: 0;
+        border-style: solid;
+    }
+
+    &:before{
+        top: -14px;
+        border-color: transparent transparent $color-border-gray transparent;
+        border-width: 7px;
+    }
+
+    &:after{
+        top: -12px;
+        border-color: transparent transparent $color-white transparent;
+        border-width: 7px;
+    }
+
+    .dropdown-item {
+        font-size: 12px;
+        font-weight: 600;
+        padding: 6px 16px;
+
+        display: flex;
+        align-items: center;
+
+        &:hover{
+            background-color: $color-gray-2;
+        }
+
+        .icon {
+            font-size: 20px;
+            font-weight: 600;
+            margin-right: 5px;
+        }
+    }
+}
+
+.btn.dropdown-toggle {
```

Comment:
@rehanabbasi I added this in the asset scss, because the change made in buttons.scss applied to our nav menu style globally. Leaving this here as a reminder to check that your intended style for padding is good to go on the asset page buttons.